### PR TITLE
EZP-30214: Add Content Tree to Bookmarks, Trash & Search

### PR DESCRIPTION
**Pull request by tischsoic**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-30214
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

## Screenshots
screen shot 2019-03-08 at 10 39 16

screen shot 2019-03-08 at 10 39 31

screen shot 2019-03-08 at 10 39 45

screen shot 2019-03-08 at 10 40 02

screen shot 2019-03-08 at 10 40 26

screen shot 2019-03-08 at 10 46 43

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by webhdx** [comment]
Can't we move it the container that has left sidebar in it, can we? This would simplify things a lot since we would have it only in one place. If not - we have to put it in separate file and include that file to avoid code duplication.

**Comment by dew326** [comment]
Well, this would require some rework of styling, but I think if we want to have this in multiple places there is a lot of value to put this in the sidebar.

File: src/bundle/Resources/views/admin/bookmark/list.html.twig [hunk 1]
```diff
@@ -115,6 +117,10 @@
     </div>
 {% endblock %}
 
+{% block react_modules %}
```
Comment (tischsoic) [inline src/bundle/Resources/views/admin/bookmark/list.html.twig]: I moved Content Tree container div to parent twig, but I've left JS in lower templates as I don't want to load them globally.

Comment (tischsoic) [reply src/bundle/Resources/views/admin/bookmark/list.html.twig]: If anyone has idea how to solve this problem I would readily hear it. :)

**Comment by tischsoic** [comment]
Contnet Tree container moved to left_sidebar - ping @dew326 @webhdx

**Comment by tischsoic** [comment]
ping @webhdx @dew326

**Comment by webhdx** [comment]
Please ignore above revert commit - I accidentally clicked `revert` button which created a revert branch.

**Pull request merged**